input/resources/operationdefinition-knowledgeartifact-release.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OperationDefinition xmlns="http://hl7.org/fhir">
+    <id value="KnowledgeArtifact-release"/>
+    <url value="http://hl7.org/fhir/uv/cmi/OperationDefinition/KnowledgeArtifact-release"/>
+    <version value="3.0.0"/>
+    <name value="KnowledgeArtifactRelease"/>
+    <title value="Knowledge Artifact Release"/>
+    <status value="draft"/>
+    <kind value="operation"/>
+    <date value="2023-01-27T09:29:23+11:00"/>
+    <publisher value="HL7 (FHIR Project)"/>
+    <contact>
+        <telecom>
+            <system value="url"/>
+            <value value="http://hl7.org/fhir"/>
+        </telecom>
+        <telecom>
+            <system value="email"/>
+            <value value="fhir@lists.hl7.org"/>
+        </telecom>
+    </contact>
+    <description value="The release operation supports updating the status of an existing draft artifact to active. The operation sets the date element of the resource and pins versions of all direct and transitive references and records them in the program's manifest. Child artifacts (i.e. artifacts of which the existing artifact is composed) are also released, recursively."/>    
+    <code value="knowledge-artifact-release"/>
+    <comment value="The release operation supports the ability of an authoring repository to transitively resolve component and dependency references, ensuring that version-specific references are determined and recorded and updating component resources to transition them to a released state. \n\nReferences MUST either be version-specific or, if they are not, an entry must exist in the version manifest to specify which version of the referenced resource should be used. The release operation SHALL ensure that the determined versions are recorded in the version manifest. If a reference is not version-specific and no entry exists in the version manifest for the referenced resource, the program is considered to be in an invalid state and not eligible for release. \n\nThe operation SHALL update the status of all components to 'active'. For both components and dependencies, if versions are not specified in the relevant reference, the operation will lookup the version to be used in the version manifest."/>
```

Comment:
What I am trying to say here though is different from that in that I was intentionally abandoning the the idea that we attempt to infer the version by looking up the "most recent version that is known" - that's why I went with the constraint that the reference has to either be version-specific or an entry has to exist in the manifest so that we can look it up. We won't try to do any version inference. Do we need to revert that thinking and plan on doing the inference? Is "most recent" determined by looking at the artifact'd date?